builtin/providers/aws/opsworks_layers.go

```diff
@@ -377,6 +392,7 @@ func (lt *opsworksLayerType) Update(d *schema.ResourceData, client *opsworks.Ops
 		CustomInstanceProfileArn:    aws.String(d.Get("custom_instance_profile_arn").(string)),
 		CustomRecipes:               lt.CustomRecipes(d),
 		CustomSecurityGroupIds:      expandStringSet(d.Get("custom_security_group_ids").(*schema.Set)),
+		CustomJson:                  aws.String(d.Get("custom_json").(string)),
```

Comment:
What happens here if CustomJson is not defined? I think it will have to change to being outside the struct declaration and then check if it is defined

e.g. https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_opsworks_stack.go#L318

Thoughts?